posts/inside-rust/2022-04-13-imposter-syndrome.md

```diff
@@ -0,0 +1,114 @@
+it's just not great for mental health, 0/10, would not recommend.
+
+The thing is though, imposter syndrome is a logical fallacy[^2]. Imposter
+syndrome occurs when we discount what we know and inflate what we think other
+people know, and this effect is often then reinforced by systemic bias for
+those of us who don't get the assumption of competence.
+
+![picture of imposter syndrome, left side shows a large circle saying "What I think others know" and a small circle inside of it saying "What I know", right side shows the same small circle saying "What I know" surrounded by many other equally sized small circles labeled "What others know"](/static/images/2022-04-13-imposter-syndrome/imposter_syndrome.jpg)
+
+In reality, we're all specialists within the Rust project. We all have areas
+where we have deep expertise and other large areas where we only have the
+vaguest idea of how things work. [Niko](https://github.com/nikomatsakis), one
+of the lang team co-leads, former compiler team lead and core team alumni,
+still comes to me to ask questions about error handling. I frequently need to
+tell my fellow contributors that I have no idea what the acronyms they're using
+mean[^3]. But this doesn't mean we don't deserve our positions within the
+project. We don't expect every contributor know everything, to be perfect, or
+to make no mistakes. The only thing we expect from our contributors is the
+ability to collaborate effectively with others and a willingness to learn and
+grow over time.
+
+The thing that makes the Rust project as good as it is isn't a couple of
+prolific contributors lifting mountains by themselves, it's everyone working
+together that brought us to where we are today. We all make mistakes. The
+project has layer[^4] after layer[^5] of safeguards to make sure we have a
+chance to catch and fix them before they affect our users. These incidents are
+unavoidable, expected, and honestly fine! This is the most fundamental
+philosophy of both the Rust language and the Rust project: we don't think it's
+sufficient to build robust systems by only including people who don't make
+mistakes; we think it's better to provide tooling and process to catch and
+prevent mistakes. It isn't an accident that our motto is "A language empowering
+everyone to build reliable and efficient software." We want people to feel
+empowered to make changes they're not 100% confident in, to make mistakes, to
+learn, and to grow within the Rust project. This is how all of us got to where
+we are today!
+
+So, if you look up to people within the Rust project, if the work we do here
+interests you, if you have always wanted to contribute, and _especially_ if you
+already have contributed, I want you to know that you're one of the people we
+want to apply for [Rust Foundation grants and
+fellowships](https://foundation.rust-lang.org/grants/). You're one of the
+people we want to eventually see join teams. You're not an imposter, and I want
```

Comment:
> You're one of the people we want to *eventually see join* teams.

(emphasis mine) 

I'd be careful with this. I'm not sure that there aren't people this post can address and help who aren't already on a team. Imposter syndrome can even impact "insiders". Let's not forget them as well!